docs/build/hosted-lotus-node.md

```diff
@@ -0,0 +1,647 @@
+
+  ```shell with-output
+  #Check if the node sync to the highest tipset
+  lotus sync wait
+  ```
+
+  ```shell
+  Worker: 10161; Base: 778820; Target: 778820 (diff: 0)
+  State: complete; Current Epoch: 778820; Todo: 0
+  
+  Done!
+  ```
+
+- Check your wallet balance
+
+  ```shell with-output
+  #Check wallet balance
+  lotus wallet list 
+  ```
+
+  ```shell
+  Address                                    Balance        Nonce  
+  t154xvuihhicgluafwmohwzwmtmqp44pwwgewyvma  100 FIL          0
+  ```
+
+### Create a node.js project
+
+Let's also set up a Node.js project with necessary dependencies, we will write basic javascript functions to focus more on learning the code to interact with Filecoin Network. 
+
+1. Create a new node.js project
+
+   ```shell
+   mkdir build-with-lotus && cd build-with-lotus
+   npm init #Initialize an project
+   ```
+   
+2. Install the fielcoin.js library
+
+   ```shell
+   npm install filecoin.js
+   ```
+
+
+### Test a simple function call
+
+At this step, you are ready to write code to connect to your Lotus node and interact with the Lotus APIs. Let's write a simple script to query the Lotus version to verify if everything works as expected.
+
+1. Let's create a `chainDataQuery.mjs` file in your project and add the following codes. We use `mjs` so we can use import statement in our code. 
+
+    ```javascript
+    //1. Import necessary packages
+    import { HttpJsonRpcConnector, LotusClient } from "filecoin.js";
+
+    //2. Use the local node URL to create a connection to your Lotus node
+    const localNodeUrl = "http://127.0.0.1:1234/rpc/v0";
+    const localConnector = new HttpJsonRpcConnector({ url: localNodeUrl });
+
+    //3. lotusClient exposes all Lotus APIs
+    const lotusClient = new LotusClient(localConnector);
+    const version = await lotusClient.common.version();
+    console.log(version);
+    ```
+
+2. Run the following command in your project directory to test it.
+
+    ```shell with-output
+    ~/build-with-lotus $ node basicQuery.mjs
+    ```
+
+    ```shell
+    {
+      Version: '1.15.0+calibnet+git.0ac1bbc7a',
+      APIVersion: 66816,
+      BlockDelay: 30
+    }
+    ```
+
+If everything is set up correctly, our code will be able to connect to the Lotus node and request chain data back as shown above. And we are ready to explore more features that the Lotus provides via API calls. 
+
+## Query chain data
+
+Blockchain data is the fundamental part of any blockchain network, it is essential that Lotus provides APIs to query Filecoin chain data, including tipSets,  blocks, and messages, network status, storage deals and storage provider's details, etc. 
+
+As one of the API client libraries, filecion.js wrapped all the Lotus APIs into simple method calls to provide  a complete library to interact with any Lotus node. Let's look at some common packages of LotusClient so you can have a sense of what you can use. 
+
+- `LotusClient.common` - methods to get node info.
```

Comment:
remove all the periods at the end of lines 165-171 (in lists, skip the periods if it's not a complete sentence)